lib/supavisor/handlers/proxy/db.ex

```diff
@@ -110,6 +123,19 @@ defmodule Supavisor.Handlers.Proxy.Db do
 
   ## Internal functions
 
+  @spec maybe_active_once(map) :: non_neg_integer()
+  def maybe_active_once(%{db_send: 0} = data) do
+    HandlerHelpers.setopts(data.db_sock, active: true)
```

Comment:
Why use `active: true`? I think it could create issues related to the messages overload with no back pressure on the sender.

Reply:
this pr is for debugging